index.js

```diff
@@ -41,7 +55,22 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+/*input - []
+output - highest metascore - number
+plan - iterate - loop thru all metascore of movies - compare and return highest number
+*/
+function getHighestMetascore(movies) {
+  if (!movies || !movies.length) {
+    return 0;
+}
+let highMetaMovie = movies[0].metascore;
+for(let i = 0; i < movies.length; i++){
+  if (movies[i].metascore > highMetaMovie) {
+    highMetaMovie = movies[i].metascore;
+  }
+}
+return parseInt(highMetaMovie);
+}
```

Comment:
I recommend using `Number(highMetaMovie)` over `parseInt()` as you may lose decimals with `parseInt()`